### ISSUE TITLE
Symbol für Gebärdensprachdolmetschen / Symbol for sign language interpretation 

### ISSUE THREAD

**Issue by linux-lukas** [hfg-gmuend/openmoji#459]
🇩🇪 

Mir ist aufgefallen, dass das Zeichen, das Menschen, die auf die (deutsche) Gebärdensprache angewiesen sind, hilft, Menschen zu finden, die sie verstehen können, noch nicht im OpenMoji Katalog ist. Ich möchte das gerne vorschlagen - hier ist ein Beispiel, wie dieses Zeichen aussieht:

🏴󠁧󠁢󠁥󠁮󠁧󠁿 / 🇺🇸 
I noticed that the sign, which helps people who rely on (German) sign language to find people who can understand them, is not yet in the OpenMoji catalog. I would like to suggest that - here is an example of what this sign looks like:

![image](https://github.com/hfg-gmuend/openmoji/assets/47597136/813492ae-b987-4d42-8ff2-70c23df246de)

**Comment by b-g** [comment]
Hallo @Lukas2112! Many thanks for the nice suggestion! We will look into it ... but it might take some time until this is done. The todos list is already long :) 

@dnlutz ok? If so ... could you that loop in and add it to our summer sprint?

**Comment by dnlutz** [comment]
Good suggestion, we'll try to include it in our next update.
I'll refine it a little bit to remove some of the black lines.

**Comment by dnlutz** [comment]
If you show us how they should look like we can give it a try ; )

**Comment by b-g** [comment]
@Lukas2112 Yes. Many thanks for this. Just the overlapping lines are not so great. Would be good to redraw those "problemzonen" areas with the same approach as e.g. the clapping hands.

 ![271855920-7c95e58c-63bd-480a-bc63-1ab12aa71de8](https://github.com/hfg-gmuend/openmoji/assets/480224/61704c89-7a71-4d23-ad95-ee6004ddec4d)

![255328823-39a32606-910e-4217-ae9d-d9a45ce8bf3c](https://github.com/hfg-gmuend/openmoji/assets/480224/12478a2d-885c-4254-b2f5-afbee2122ac1)
